### DISCUSSION TITLE
Run in background

### DISCUSSION THREAD

**Discussion by Elmatbaisaad** [symfony/symfony#46521] [Q&A] [answered]
Hello everyone .
i Have a Route the only work that do is extract data from a server i need to run it in backround even i close the browser that make me extract data continuously .
**Please How can do that ?**
Thank you !

**Comment by stof** [comment, marked as answer]
This is a use case for doing the work in an actual background process, by using the Messenger component. The controller would then only dispatch the message in the Messenger bus.
